azure-kusto-data/tests/aio/test_async_token_providers.py

```diff
@@ -351,13 +350,9 @@ async def inner():
     @aio_documented_by(TokenProviderTests.test_azure_identity_default_token_provider)
     @pytest.mark.asyncio
     async def test_azure_identity_token_provider(self):
-        app_id = os.environ.get("APP_ID", "b699d721-4f6f-4320-bc9a-88d578dfe68f")
-        os.environ["AZURE_CLIENT_ID"] = app_id
-        auth_id = os.environ.get("APP_AUTH_ID", "72f988bf-86f1-41af-91ab-2d7cd011db47")
-        os.environ["AZURE_TENANT_ID"] = auth_id
-        app_key = os.environ.get("APP_KEY")
-        os.environ["AZURE_CLIENT_SECRET"] = app_key
-
+        app_id = get_app_id()
```

Comment:
Mind these can now be None

Reply:
How? if they don't exist it throws
I don't think you can set an empty env var

Reply:
Meaning these had defaults which you omitted from the code

Reply:
Yes, these are defaults in regard to our specific E2E test cluster. Which is very weird, and will fail for anyone who doesn't have the creds for it.
It will continue to work on github because the env is set properly, and also for anyone who configured their e2e tests.